packages/clerk-js/src/ui/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -20,14 +20,15 @@ const createSelectOption = (country: CountryEntry) => {
 
 const countryOptions = [...IsoToCountryMap.values()].map(createSelectOption);
 
-type PhoneInputProps = PropsOfComponent<typeof Input>;
+type PhoneInputProps = PropsOfComponent<typeof Input> & { defaultSelectedIso?: CountryIso };
```

Comment:
🙃 This is extreme nitpicking, you can ignore it
```suggestion
type PhoneInputProps = PropsOfComponent<typeof Input> & { defaultSelectedISO?: CountryIso };
```

Reply:
Even more nitpicky (sorry for this hehe) I think that at some point we landed on using camelCase for abbreviations as the newer w3c standard/airbnb guide suggests, eg:

✅  `redirectUrl` 
❌  `redirectURL`